slave/benchmarks_remote.py

```diff
@@ -61,7 +62,8 @@ def translate_path(path):
 
     @staticmethod
     def inject_data(path, data):
-        print "inject"
+        logger = logging.getLogger('Octane.inject_data')
+        utils.log_info(logger, "inject")
```

Comment:
Feel free to remove these two lines, they're not useful at all (I think it was debug prints from the time where the proxy was being programmed)

Reply:
Re: Windows. Thanks! Totally missed that.

All of the print "inject" stuff? Great!